src/spec/apis/funds_api.py

```diff
@@ -142,10 +144,24 @@ async def list_historical_values_spec(
     ) -> List[HistoricalValue]:
         """Lists historical values"""
         return await self.list_historical_values(
-            fundId,
+            self.toUuid(fundId),
             first_result,
             max_results,
             start_date,
             end_date,
             token_bearerAuth
         )
+
+    def toUuid(self, str: str) -> UUID:
```

Comment:
missed this one. please changfe str to a proper paramaeter name. also in docstring.